steps/previsit/start.tsx

```diff
@@ -24,11 +22,11 @@ const step = {
     submit: (nextSlug?: string): ReturnType<typeof makeSubmit> =>
       makeSubmit([
         {
-          url: urlObjectForSlug(nextSlug),
+          slug: nextSlug as PageSlugs | undefined,
```

Comment:
I'm getting `Warning: Encountered two children with the same key, `/thc/6e814765-d253-ea11-a813-000d3a7f1b13/about-visit` (and `/start`) because the submits are going to the same place.

Reply:
Fixed this by choosing a different slug for the placholder.

Reply:
Fixed this by choosing a different slug for the placeholder.